### DISCUSSION TITLE
CL constraint for the MACH tutorial wing

### DISCUSSION THREAD

**Discussion by AbhijithMoni** [mdolab/dafoam#678] [General]
Hi, 

I have a question regarding the aerostructural tutorial (MACH tutorial wing) for `runScript_AeroStruct.py`. In the problem description, it is stated that the **Lift coefficient (CL) is expected to be 0.5**. However, in the tutorial, the CL constraint is defined as follows:

```python
self.add_constraint("cruise.aero_post.CL", equals=0.3, scaler=1.0)
```

As a result, the solver converges to a CL of 0.3, not 0.5 as described. 

Could you please clarify whether this discrepancy is intentional for the tutorial case, or if it might be an error in the script?

Many thanks in advance.

Best regards,  
Abhijith

![image](https://github.com/user-attachments/assets/6a237700-1cde-4bd5-84f6-185fb8b174b3)

**Comment by friedenhe** [comment]
Thanks for catching this bug. We should have set this line as:
`self.add_constraint("cruise.aero_post.CL", equals=CL_target, scaler=1.0)`
This issue has been fixed.

**Comment by AbhijithMoni** [comment]
@friedenhe Thank you for addressing the issue and taking the time to implement the fix.
